src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -148,7 +148,7 @@ internal override void PostPersist(ApplicationEngine engine)
             var committee = GetCommitteeFromCache(engine.Snapshot);
             var pubkey = committee.ElementAt(index).PublicKey;
             var account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
-            GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100, false);
+            _ = GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100, false);
```

Comment:
this call should be awaited. Since it's void return, we need a try/catch block as well